homeassistant/components/aladdin_connect/manifest.json

```diff
@@ -1,11 +1,10 @@
 {
   "domain": "aladdin_connect",
   "name": "Aladdin Connect",
-  "codeowners": ["@mkmer"],
+  "codeowners": ["@ajohnston1219"],
```

Comment:
Why are you adding `@ajohnston1219` as code owner and not yourself?

Reply:
Nevermind, I think I managed to get this to work this time.